external/virtualization-layer/recipes-containers/docker/docker-ce_%.bbappend

```diff
@@ -1,11 +1,11 @@
-do_install:append() {
+do_install:append:tegra() {
     if ${@bb.utils.contains('DISTRO_FEATURES', 'sysvinit', 'true', 'false', d)};    then
         install -d ${D}${sysconfdir}/init.d
         install -m 0755 ${WORKDIR}/docker.init ${D}${sysconfdir}/init.d/docker.init
     fi
 }
 
-RRECOMMENDS:${PN}:append = " \
+RRECOMMENDS:${PN}:append:tegra = " \
     kernel-module-br-netfilter \
     kernel-module-esp4 \
     kernel-module-ip-vs \
```

Comment:
With the above changes, we'll also need
```
PACKAGE_ARCH:tegra = "${TEGRA_PKGARCH}"
```

Reply:
I will make it arch specific

Reply:
This append was what caught my attention and then I took the opportunity to correct the others